src/fusion-rates-gamow.md

```diff
@@ -0,0 +1,861 @@
+
+$$V_{mol}(r) = \frac{2}{r}(1 - b_1r - b_2r^2) e^{-\alpha r^s}$$
+
+with $r$ in units of the Bohr radius ($a_0$), $V_{mol}$ is in Rydbergs and with
+- $\alpha = 0.6255121237003474$
+- $b_1 = 1.4752572117720800$
+- $b_2 = -0.2369829512108492$
+- $s = 1.0659864120418940$
+
+We'll make a function that aligns with the nuclear potential in the sense that it takes $r$ in fm and returns the potential in MeV.
+
+```python
+alpha = 0.6255121237003474
+b1 = 1.4752572117720800
+b2 = -0.2369829512108492
+s = 1.0659864120418940
+
+# Kolos potential in MeV with r in fm
+def V_mol(r_fm):
+    r = r_fm*1e-15/a0 # fm to a0
+    return (2 / r) * (1 - b1 * r - b2 * r**2) * np.exp(-alpha * r**s) * Ry_to_eV / 1e6
+```
+
+```python
+# Range for r in pm for plotting and fm for the potential functions
+r_pm = np.linspace(25, 500, 500)
+r_fm = r_pm*1000
+
+# Calculate V_mol in MeV
+V_mol_values = V_mol(r_fm)
+
+# Plot
+plt.figure()
+plt.plot(r_pm, V_mol_values*1e6, label=r"$V_{mol}(r)$")
+plt.xlabel(r"$r$ (pm)")
+plt.ylabel(r"$V_{mol}$ (eV)")
+# plt.savefig("mol-potential-D2.png", dpi=600)
+plt.show()
+```
+
+We can see how much energy the deuterons have in their equilibrium position of 74 pm:
+
+```python
+V_mol(74000)*1e6
+```
+
+To put this approximately $-5 \, \rm eV$ into perspective, we need to compare it with the energy that two deuterons would have in the absence of any binding electrons - i.e. the "bare" Coulomb potential. 
+
+
+### Coulomb potential
+
+Without the electrons the electrostatic potential between the deuterons is given by:
+
+$$V_{\rm coulomb} = \frac{e^2}{4\pi\epsilon_0r}$$
+
+```python
+# Coulomb potential in MeV with r in fm
+def V_coulomb(r_fm):
+    return e**2/(4 * np.pi * eps0 * r_fm*1e-15) * J_to_eV / 1e6
+```
+
+Let's plot $V_{\rm coulomb}$ and $V_{\rm mol}$ on the same scale for easy comparison.
+
+```python
+# Range for r in pm for plotting and fm for the potential functions
+r_pm = np.linspace(25, 500, 500)
+r_fm = r_pm*1000
+
+# Calculate V_mol in MeV
+V_coulomb_values = V_coulomb(r_fm)
+
+# Calculate V_mol in MeV
+V_mol_values = V_mol(r_fm)
+
+# Plot
+plt.figure()
+plt.plot(r_pm, V_mol_values*1e6, label=r"$V_{mol}$")
+plt.plot(r_pm, V_coulomb_values*1e6, label=r"$V_{coulomb}$")
+plt.xlabel(r"$r$ (pm)")
+plt.ylabel(r"$V}$ (eV)")
+plt.axvline(x=74, color="grey", linestyle='--', linewidth=1)
+plt.legend()
+# plt.savefig("mol-coulomb-potential-D2.png", dpi=600)
+plt.show()
+```
+
+Although the effect of the electrons is in general complicated, we can see that overall the electrons act to lower the bare Coulomb potential. If we take 74 pm as a representative separation of the deuterons in the molecule, then we can roughly say that the electrons have lowered the coulomb barrier by an amount:
```

Comment:
In the main text I would always capitalize the word "Coulomb" for consistency (not in the code)